api.planx.uk/editor/publish.test.ts

```diff
@@ -184,26 +258,26 @@ it("updates published flow and returns altered nodes if there have been changes"
 const mockFlowData: Flow["data"] = {
```

Comment:
Thanks for adding node names, that's helpful. 
nit: Could this data live in its own file to make the test file more focused?